client/app/containers/tests-runs/tests-runs-search/tests-runs-search.html

```diff
@@ -70,4 +63,11 @@
             <button ng-show="$ctrl.isSearchActive()" class="tests-runs-search__reset-btn md-button md-ink-ripple pull-right" type="button" ng-click="$ctrl.onReset();"><span>RESET</span></button>
         </div>
     </div>
+    <md-fab-speed-dial id="main-fab" ng-if="$ctrl.isMobile() && !$ctrl.isFilterActive()" md-direction="up" class="md-scale md-fab-bottom-right">
+        <md-fab-trigger>
+            <md-button id="mainFab" aria-label="menu" class="md-fab md-primary" ng-click="$ctrl.showSearchDialog($event)" md-visible="tooltipVisible">
+                <div class="search-fab" ng-class="{' _list': !$ctrl.isModalSearchActive(), '_checked-list': $ctrl.isModalSearchActive()}" aria-hidden="true"></div>
```

Comment:
do you need block type element here? Maybe `span` is better choice for this place?

Reply:
yes, because we use svg image as bg instead of font awesome elements